src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -475,6 +475,11 @@ private void CompileNamedType(NamedTypeSymbol containingType)
                 if (compilationState.Emitting)
                 {
                     CompileSynthesizedExplicitImplementations(sourceTypeSymbol, compilationState);
+
+                    if (containingType.IsExtension)
+                    {
+                        CompileSynthesizedExtensionMarker((SourceExtensionTypeSymbol)containingType, compilationState);
```

Comment:
>CompileSynthesizedExtensionMarker((SourceExtensionTypeSymbol)containingType, compilationState); [](http://example.com/codeflow?start=24&length=95)

Do we want to add it before SynthesizedExplicitImplementations?

Reply:
I couldn't see a reason to change the order here. What did you have in mind?